themes/digital.gov/src/scss/new/_topics-button.scss

```diff
@@ -18,7 +18,8 @@
     white-space: nowrap;
 
     @include at-media("tablet") {
-      padding-left: units(1);
+      padding-left: 6px;
+      padding-right: 6px;
```

Comment:
There's a `px-to-rem()` mixin you can use.

https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/functions/units/px-to-rem.scss

Reply:
updated to use the px-rem mixin